Tests/MMEMetricsManagerTests.m

```diff
@@ -60,11 +60,11 @@ - (void)testAttributesExcludesNullIsland {
 }
 
 - (void)testAttributesDoesntExcludesNullFloat {
-    CLLocation *notNullIsland = [[CLLocation alloc] initWithLatitude:0.4 longitude:0.2];
+    CLLocation *notNullIsland = [[CLLocation alloc] initWithLatitude:4 longitude:2];
```

Comment:
Problem:
updateCoordinate filters out coordinates that are `close` to null island.

Because the test coordinates are < 0.5, they are rounded to (0.0, 0.0) leaving the test assuming it wasn’t updated.

Change:
Bump to coordinates (4,2) which will not be rounded down to 0,0